### ISSUE TITLE
Deploy on Windows not working

### ISSUE THREAD

**Issue by stevenspinoza** [christopherwxyz/NetSuiteSDF#43]
Hey, thanks for the dev-friendly tool. Although, I did the setup and could import custom records/files to the project, I couldn't manage to deploy them. The console does show any error/success message when I try to deploy. Thanks.

PD: I'm using VSCODE on Windows 8.1
PD2: I did use the last resort, the plain sdfcli, and it worked. Hopefully, not long before you guys find this bug. Thanks again.

**Comment by JonnyBoy333** [comment]
Are you working from the master branch in this repository or from the version in the vscode marketplace? If you're working from the master branch do you have "quickdeploy" checkbox checked in the extension's preferences (I do not believe this is available in the marketplace version)? Quick deploy does not work on windows yet, some of the paths need to be updated.

**Comment by stevenspinoza** [comment]
Thanks @JonnyBoy333 for the reply. Yes, the Quickdesploy option was checked. I installed it from the extensions sidebar (I guessed it is the same from the marketplace version). I unchecked it and it started logging ~~until after I typed YES in the prompt about deploying to production. After that it logged Cancelling deployment and stopped.~~ My bad, I had to type "**Deploy**"!!

Thank you, hopefully that's the last issue I encounter using NSSDF. :thumbsup::thumbsup:
